packages/build-info/tests/get-build-info.test.ts

```diff
@@ -29,7 +29,6 @@ describe('Golang', () => {
     })
     expect(info).toMatchInlineSnapshot(`
       {
-        "buildSystems": [],
```

Comment:
I would return an empty array if no build systems could be detected

Reply:
`buildSystems` in an optional property. We mark it as **unknown** in Buildbot if it is undefined.

Reply:
Yeah, if no build system detected an empty list is returned.